### ISSUE TITLE
Fix and re-enable TestCDATASection - theme related

### ISSUE THREAD

**Issue by KirillOsenkov** [mono/monodevelop#5322]
https://github.com/mono/monodevelop/blob/d9dd699478a1fd387c24ef6604c40ee300185ea5/main/src/core/MonoDevelop.TextEditor.Tests/Mono.TextEditor.Tests/SyntaxHighlightingTests.cs#L122

Disabling the test that fails on CI:

```
expected:<span foreground="#e5da73">&lt;![CDATA[ test]]&gt;</span>
But got:<span foreground="#eeeeec">&lt;![CDATA[ test]]&gt;</span>
String lengths are both 57. Strings differ at index 20.
Expected: "<span foreground="#e5da73">&lt;![CDATA[ test]]&gt;</span>"
But was: "<span foreground="#eeeeec">&lt;![CDATA[ test]]&gt;</span>"
-------------------------------^
```

Need to reenable and fix such that the test doesn't depend on current theme, but is deterministic and reproducible.

See related PR https://github.com/mono/monodevelop/pull/5291 where the test failed.

> VS bug [#643382](https://devdiv.visualstudio.com/DevDiv/_workitems/edit/643382)

**Comment by KirillOsenkov** [comment]
Mike fixed this as part of https://github.com/mono/monodevelop/pull/5291

**Comment by xamarin-release-manager** [comment]
Fixed in version 7.7.0.522 (master)

Author: Kirill Osenkov
Commit: 124ec076ca0091785a77629d38eb4371dbdf4d11 (mono/monodevelop)

**Comment by xamarin-release-manager** [comment]
Fixed in version 7.6.0.1977 (release-7.6)

Author: Kirill Osenkov
Commit: 9abe324bd3e2883d268ef56053fca139c6caf93f (mono/monodevelop)

**Comment by xamarin-release-manager** [comment]
Fixed in version 8.6.0.802 (master)

Author: Kirill Osenkov
Commit: 124ec076ca0091785a77629d38eb4371dbdf4d11 (xamarin/monodevelop)
